yarn.lock

```diff
@@ -3181,6 +3172,17 @@
     "@types/yargs" "^15.0.0"
     chalk "^4.0.0"
 
+"@jest/types@^27.0.6":
+  version "27.0.6"
+  resolved "https://registry.npmjs.intuit.com:443/artifactory/api/npm/npm-intuit/@jest/types/-/types-27.0.6.tgz#9a992bc517e0c49f035938b8549719c2de40706b"
```

Comment:
This is also pulling from the `intuit` registry -- you can checkout the `master` `yarn.lock` file and re-run `yarn` with the changes to the `.npmrc` above to pull from the public one